functional/verifier_registrar_unreachable/test.sh

```diff
@@ -0,0 +1,119 @@
+            rlRun "limeUpdateConf verifier challenge_lifetime 1800"
+            rlRun "limeUpdateConf agent attestation_interval_seconds 10"
+            # Right now just for push agent
+            rlRun "limeUpdateConf agent exponential_backoff_initial_delay 1000"
+        fi
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            rlRun "limeCondStartAbrmd"
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        sleep 5
+        # Need to start verifier for generating
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStopVerifier"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Try to star just agent and check behavior when other services are down"
+        rlRun "limeStart${AGENT_SERVICE}"
+        rlRun -s "limeWaitForAgentRegistration ${AGENT_ID}" 1
+        rlAssertGrep "ERROR - Agent $AGENT_ID does not exist on Registrar" $rlRun_LOG -E
+        #check agent status in logs
+        rlAssertGrep "Error.*Connection refused" $(limeAgentLogfile)
+    rlPhaseEnd
+
+    rlPhaseStartTest "Try register agent after startup registrar, without running verifier, should succeed"
+        limeTIMEOUT=80
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+        rlRun -s "limeWaitForAgentRegistration ${AGENT_ID}" 0
+        rlAssertGrep "SUCCESS: Agent $AGENT_ID registered" $(limeAgentLogfile)
+        rlAssertGrep "SUCCESS: Agent $AGENT_ID activated" $(limeAgentLogfile)
+    rlPhaseEnd
+
+    rlPhaseStartTest "Verify that adding agent fails when verifier is unreachable"
+        limeTIMEOUT=20
+        rlRun "limeCreateTestPolicy"
+        rlRun -s "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --runtime-policy policy.json -c add ${TENANT_ARGS}" 1
+        rlAssertGrep "Failed to establish a new connection.*Connection refused" $rlRun_LOG -E
+        rlRun "limeWaitForAgentStatus $AGENT_ID 'Connection Refused'" 1
+        rlAssertGrep "GET invoked from" $(limeAgentLogfile)
+    rlPhaseEnd
+
+    rlPhaseStartTest "Start verifier, establish attestation and then make verifier unreachable again"
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"     
+        # Add the agent to the verifier database
+        rlRun "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --runtime-policy policy.json -c add ${TENANT_ARGS}"
+        rlRun "limeWaitForAgentStatus $AGENT_ID 'Get Quote'"
+        rlRun -s "keylime_tenant -c cvlist"
+        rlAssertGrep "{'code': 200, 'status': 'Success', 'results': {'uuids':.*'$AGENT_ID'" $rlRun_LOG -E
+        for i in {1..2}; do
+            rlLogInfo "Iteration $i: Stopping and Restarting Verifier"
+            rlRun "limeStopVerifier"
+            rlRun "limeWaitForAgentStatus $AGENT_ID 'Registered'"
```

Comment:
Drop this cmd. Rather Wait  10 seconds. And please, use variable for 10 secs used here and for the attestation interval config.